packages/blockchain-api/deploy.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Deploys the notification serivice to App Engine
+#
+# Flags:
+# -n: Name of the network, maps to App Engine 'service' (integration, production, etc.)
+
+NETWORK=""
+
+while getopts 'a:n:' flag; do
+  case "${flag}" in
+    n) NETWORK="$OPTARG" ;;
+    *) error "Unexpected option ${flag}" ;;
+  esac
+done
+
+[ -z "$NETWORK" ] && echo "Need to set the NETWORK via the -n flag" && exit 1;
+
+echo "Starting blockchain api deployment."
+
+echo 'Copying over config file'
+cp "config/config.${NETWORK}.env" .env
```

Comment:
See comment above. blockchain api shouldnt do this.
@sallyjyl Didn't you have a PR that removes notification service's dependency on the config files?

Reply:
It was part of the issue that I didn't end up being able to test due to npm(?) issues.
So didn't make it into a PR e=yet

Reply:
Okay, please do so we don't lose that work. It's possible we won't pick the contract kit task back up before the end of your internship